target-app/Home/Views/SaveTargetBottomSheetViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  SaveTargetBottomSheetViewController.swift
+//  target-app
+//
+//  Created by Andres Leonel Torrico Cossio on 14/09/2022.
+//
+
+import UIKit
+
+class SaveTargetBottomSheetViewController: UIViewController {
```

Comment:
Do we really need to name this after _BottomSheet_? Is it possible to use this View controller in other contexts?

Reply:
For now, we will use this VC just as a modal bottom sheet